### DISCUSSION TITLE
Inav App Icon - Show Version Number

### DISCUSSION THREAD

**Discussion by Goatspunk** [iNavFlight/inav#8993] [Ideas]
After being out in the field in full sun trying to juggle 3 versions of iNav across 3 different aircraft, I found it was a huge challenge to make sure you're using the correct iNav configurator.  We had version 2, 5, and 6.  We didn't have time to reflash, set up, and maiden the aircraft with the newest version, so that's out of the question.  The only way to see what version of iNav you have on your laptop is by looking at the tiny version number in gray on a black background in the top left corner and that's hard in the field under full sun brightness on most laptops.

Request:  Make the inav app icon have the version number on it in white so you know which instance you have open.
![inav 6 icon](https://user-images.githubusercontent.com/60055641/233161435-e4402ffa-0700-4826-be21-5646ec6a4859.png)

**Comment by MrD-RC** [comment]
Just change the text under the app icon to have the version number.
